app/assets/javascripts/app/views/translation/index.jst.eco

```diff
@@ -18,7 +18,7 @@
 
   <div class="box box--message">
     <h2><%- @T('Inline translation') %></h2>
-    <p><%- @T('To make translations easier you can enable and disable the inline translation feature by pressing "%s".', 'ctrl+alt+t') %></p>
+    <p><%- @T('To make translations easier you can enable and disable the inline translation feature by pressing ') %><%= @inlineTranslationKey%></p>
```

Comment:
Here the placeholder `%s` was removed from the translatable string. This needs to be corrected to have the possibility to translate the sentence without this hardcoded output at the end.